mypy/git.py

```diff
@@ -1,18 +1,28 @@
 """Utilities for verifying git integrity."""
 
+from __future__ import print_function
```

Comment:
You need at least a comment reminding the reader. It would also be nice if the unit tests somehow enforced this. (If someone accidentally breaks this and the tests don't catch it, it's a real problem.)

But honestly I would much prefer if the setup.py for mypy_extensions didn't depend on this module at all.